packages/clay-management-toolbar/src/ResultsBar.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayLabel from '@clayui/label';
+import ClayLink from '@clayui/link';
+import React from 'react';
+import ResultsBarItem from './ResultsBarItem';
+
+interface IProps {
+	clearLabel?: string;
+	clearResultsURL?: string;
+	filterLabels?: Array<
+		React.ComponentProps<typeof ClayLabel> & {value: React.ReactText}
+	>;
+	onClearButtonClick?: (
+		e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+	) => void;
+	searchValue?: string;
+	spritemap?: string;
+	totalItems?: number;
+}
+
+const ResultsBar: React.FunctionComponent<IProps> = ({
+	clearLabel,
+	clearResultsURL,
+	filterLabels = [],
+	onClearButtonClick,
+	searchValue,
+	spritemap,
+	totalItems,
+}) => (
+	<nav className="tbar tbar-inline-xs-down subnav-tbar subnav-tbar-primary">
+		<div className="container-fluid container-fluid-max-xl">
+			<ul className="tbar-nav tbar-nav-wrap">
+				<ResultsBarItem expand={!(filterLabels.length > 0)}>
+					<span className="component-text text-truncate-inline">
+						<span className="text-truncate">
+							{`${totalItems} results for "`}
```

Comment:
What do you think about wrap this text with a `sub`?

Reply:
Yeah, I haven't added `sub` yet, that's one of the things I said in the PR description.